tools-local/tasks/Microsoft.DotNet.SourceBuild.Tasks/WriteBuildOutputProps.cs

```diff
@@ -33,6 +34,10 @@ public class WriteBuildOutputProps : Task
         /// </summary>
         public ITaskItem[] ExtraPackageInfo { get; set; }
 
+        // Additional asset paths are assumed to have the structure <pathToAsset>/<assetName>/<assetVersion>
+        // i.e. /bin/obj/x64/Release/blobs/Toolset/3.0.100
```

Comment:
`/// <summary>` comments for intellisense/consistency?

Reply:
Added.